### PR TITLE
add documentation for info command

### PR DESCRIPTION
**Pull request by ersachin3112**
add detailed documentation for `npx webpack info` command.

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -84,6 +84,28 @@ Name of template to generate.
...
+npx webpack info --output=json
+```
+
+**Options**
+
+**`--output`**
+
+`string`
+
+To get the output in specified format (accept `json` or `markdown`).
```
Comment (snitin315) [inline src/content/api/cli.md:107]: https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types

```suggestion
`string : 'json' | 'markdown'`

To get the output in specified format.
```

**Review by snitin315 — APPROVED** [review]
Thanks

File: src/content/api/cli.md [hunk 2]
```diff
@@ -84,6 +84,28 @@ Name of template to generate.
...
+npx webpack info [options]
+```
+
+**example**
+
+```bash
+npx webpack info --output=json
+```
+
+**Options**
```
Comment (chenxsan) [inline src/content/api/cli.md:101]:
```suggestion
#### Options for info
```

**Review by chenxsan — COMMENTED** [review]
Can you document other commands like `build`, `configtest` as well now that we have documented `init` and `info`?

**Comment by ersachin3112** [comment]
> Can you document other commands like `build`, `configtest` as well now that we have documented `init` and `info`?

Sure I will do in a separate PR.

**Review by chenxsan — APPROVED** [review]

**Pull request merged**